Objective-C/CBLReplicator.mm

```diff
@@ -743,7 +736,9 @@ - (void) _resolveConflict: (CBLReplicatedDocument*)doc {
             return;
         }
     } else {
-        collection = [_config.database defaultCollectionOrThrow];
+        collection = [_collectionMap objectForKey: $sprintf(@"%@.%@",
```

Comment:
There is no need to have if/else anymore because doc.collection will not be nil unless there is a bug. Will be just :

```
CBLCollection* collection = [_collectionMap objectForKey: $sprintf(@"%@.%@", doc.scope, doc.collection)];
Assert(collection, "Collection not found in replicator config when resolving a conflict");
```